cmake/Modules/FindQuadMath.cmake

```diff
@@ -0,0 +1,51 @@
+# Module that checks whether the compiler supports the
+# quadruple precision floating point math
+#
+# Sets the following variables:
+# HAVE_QUAD
+# QUADMATH_LIBRARIES
```

Comment:
No. And I wonder how the linking will work for downstream modules without QUADMATH_LIBARIES. Thought this was the magic part of our build system. Is it maybe linked to libopmcommon and hence picked up automatically (must have missed that)?

Reply:
It is only used in two simulatorsn in opm-models, see the downstream pr, think it will be clearer then.